jetty9-base/src/main/jetty-base/etc/java-util-logging.properties

```diff
@@ -0,0 +1,9 @@
+.level=INFO
+
+handlers=com.google.cloud.logging.LoggingHandler
+com.google.cloud.logging.LoggingHandler.level=FINE
+com.google.cloud.logging.LoggingHandler.log=gae_app.log
```

Comment:
I initially had the name jetty.log, but then when we added the monitored resource it is of type "gae_app",  so I thought it made more sense to have the type and the filename match.

Note also that the file will contain logging from anything that uses JUL as well as the jetty logs that are redirected to JUL.   So it can contain both container and application logging.